plugins/BEdita/API/src/Controller/ResourcesController.php

```diff
@@ -226,6 +228,11 @@ public function resource($id)
         $entity = $action(['primaryKey' => $id, 'contain' => $contain]);
 
         if ($this->request->is('delete')) {
+            // If object is not deletable returns 403 FORBIDDEN
+            if ((in_array($this->modelClass, ['Roles', 'Users'])) && ((int)$id === 1)) { // ID 1 not removable
```

Comment:
This way it works, but I don't know if this check should be performed inside DeleteEntityAction to make sure we have same behavior in other possible use cases

Reply:
Fair enough